doc/nrf/protocols/matter/getting_started/custom_clusters.rst

```diff
@@ -102,23 +103,78 @@ See the description of each element in the following tabs:
+            <command source="client" code="0x02" name="MyCommand" response="MyCommandResponse" optional="false">
+               <description>Command that takes two uint8 arguments and returns their sum.</description>
+               <arg name="arg1" type="int8u"/>
+               <arg name="arg2" type="int8u"/>
+            </command>
+            <event source="server" code="0x01" name="MyEvent" optional="false">
+               <description>Event that is generated by the server.</description>
+               <arg name="arg1" type="int8u"/>
+            </event>
          </cluster>
 
+   .. tab:: ``<cluster>``
+
+      ``<clusterExtension>`` defines the extension of an existing cluster and consist of the following attributes:
+
+      * ``code`` - A 32-bit identifier for the existing cluster, that should be extended.
+      * ``<attribute>`` - An attribute definition within the cluster.
+
+         * ``side`` - Specifies whether the attribute is on the client or server side.
+         * ``code`` - A unique identifier for the attribute within the cluster.
+         * ``define`` - The C++ preprocessor macro name for the attribute, typically in uppercase with words separated by underscores.
+         * ``type`` - The data type of the attribute.
+         * ``entryType`` - The data type of array elements if the attribute is an array.
+         * ``writable`` - Indicates whether the attribute can be written to.
+         * ``default`` - The default value of the attribute.
+         * ``optional`` - Indicates whether the attribute is optional.
+         * ``name`` - The name of the attribute.
+
+      * ``<command>`` - A command definition within the cluster.
+
+         * ``source`` - Specifies whether the command originates from the client or server.
+         * ``code`` - A unique identifier for the command within the cluster.
+         * ``name`` - The name of the command.
+         * ``optional`` - Indicates whether the command is optional.
+         * ``disableDefaultResponse`` - Indicates whether the default response to the command is disabled.
+         * ``response`` - The name of the response command, if any.
+         * ``description`` - A brief description of the command's purpose and functionality.
+         * ``arg`` - An argument for the command, specifying its name and type.
+
+      * ``<event>`` - An event definition within the cluster.
+
+         * ``source`` - Specifies whether the event originates from the client or server.
+         * ``code`` - A unique identifier for the event within the cluster.
+         * ``name`` - The name of the event.
+         * ``optional`` - Indicates whether the event is optional.
+         * ``description`` - A brief description of the event's purpose and functionality.
+         * ``arg`` - An argument for the event, specifying its name and type.
+
+      For example, the following XML code extends a Basic information cluster with one attribute, one command, and one event:
```

Comment:
Do we need to have Basic with capital B?

Reply:
actually I think both B and I should be capitalized as this is cluster's name: https://github.com/project-chip/connectedhomeip/blob/master/src/app/zap-templates/zcl/data-model/chip/basic-information-cluster.xml#L68